source/UART_API.c

```diff
@@ -0,0 +1,49 @@
+/*
+* File:   UART_API.h
+* Author: Marco Jost
+* Comments:  This Module implements the UART API. The Module contains functions for initialization and sending characters
+* Revision history:  1.0
+*/
+
+#include <xc.h>
+
+#include "log.h"
+#include "global.h"
+#include "UART_API.h"
+
+
+void uart_init(void) {
+
+	U1MODEbits.BRGH = 0;
+	U1BRG = ((PERIPHERAL_FREQUENCY / 16 / BAUDRATE) - 1);
+
+	TRISBbits.TRISB14 = 0;	//Set Pin14(TX) as output
+	TRISBbits.TRISB15 = 1; //Set Pin15(RX) as (analog) input
+
```

Comment:
Please change to predefined definitions from global.h
UART_TX_TRIS
UART_RX_TRIS